apps/meteor/client/sidebar/sections/components/OmnichannelMakeCall.tsx

```diff
@@ -0,0 +1,31 @@
+import { Sidebar } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { ReactElement } from 'react';
+
+import { useOutboundDialer } from '../../../../ee/client/hooks/useOutboundDialer';
+
+export const OmnichannelMakeCall = (): ReactElement => {
```

Comment:
This component is not necessary as it will be replaced by the Dial Pad being implemented by @tiagoevanp

Reply:
Agreed. This is just to demonstrate the usage. Even the changes below will be removed once the dialpad is using this.